### DISCUSSION TITLE
what is calculatornuxt ?

### DISCUSSION THREAD

**Discussion by slow-sheep** [nuxt/nuxt#26833] [General]
my project has a problem.  
like a ssr error, but icant see reason. in my log systeam. no used infomation, i just know ### **error page request nuxt file-path is calculatornuxt,** is no nomoal file-path,  and server/middlware/log.ts new request console.log no do.
this is error img:
![(8VFN{K)J $JHK%06@NOI%U](https://github.com/nuxt/nuxt/assets/117260282/d49ab385-4dbf-4c9c-81d7-f77b9be64065)

**Comment by huang-julien** [comment]
it looks like you've set a baseURL

**Reply by slow-sheep** [reply]
thanks! give  u chinese best review: "牛逼"
